[2024-05-08T00:28:15.403+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-08T00:28:15.447+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1_monitoring.tarea2 scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-05-08T00:28:15.471+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1_monitoring.tarea2 scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2024-05-08T00:28:15.476+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-05-08T00:28:15.509+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): tarea2> on 2024-01-02 00:00:00+00:00
[2024-05-08T00:28:15.534+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4572) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-08T00:28:15.545+0000] {standard_task_runner.py:63} INFO - Started process 4599 to run task
[2024-05-08T00:28:15.554+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '6.1_monitoring', 'tarea2', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '374', '--raw', '--subdir', 'DAGS_FOLDER/061_monitoring.py', '--cfg-path', '/tmp/tmpavql503_']
[2024-05-08T00:28:15.557+0000] {standard_task_runner.py:91} INFO - Job 374: Subtask tarea2
[2024-05-08T00:28:15.796+0000] {task_command.py:426} INFO - Running <TaskInstance: 6.1_monitoring.tarea2 scheduled__2024-01-02T00:00:00+00:00 [running]> on host 521bf890d837
[2024-05-08T00:28:16.005+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1_monitoring' AIRFLOW_CTX_TASK_ID='tarea2' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T00:00:00+00:00'
[2024-05-08T00:28:16.007+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-08T00:28:16.038+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-08T00:28:16.040+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "sleep 2 && echo 'Segunda tarea!'"]
[2024-05-08T00:28:16.053+0000] {subprocess.py:86} INFO - Output:
[2024-05-08T00:28:18.055+0000] {subprocess.py:93} INFO - Segunda tarea!
[2024-05-08T00:28:18.056+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-08T00:28:18.057+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-08T00:28:18.088+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=6.1_monitoring, task_id=tarea2, run_id=scheduled__2024-01-02T00:00:00+00:00, execution_date=20240102T000000, start_date=20240508T002815, end_date=20240508T002818
[2024-05-08T00:28:18.143+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-08T00:28:18.175+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-08T00:28:18.179+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
